examples/bootc-uki/build.base

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eux
+
+cd "${0%/*}"
+
+cargo build --release --features=pre-6.15 --bin cfsctl --bin composefs-setup-root
+
+cp ../../target/release/cfsctl .
+cp ../../target/release/composefs-setup-root extra/usr/lib/dracut/modules.d/37composefs/
```

Comment:
Hmm, what to do about this?  Fold it into `bootc internals` as well?  We had discussion recently re: binary size but I can't find it offhand...